packages/core/admin/admin/src/hooks/useFetchClient/index.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useRef } from 'react';
+import { getFetchClient } from '../../utils/getFetchClient';
+
+const cancelToken = () => {
```

Comment:
NIT, not sure if these needs to be it's own function?

Reply:
yes, I agree is just for two lines of code, I have removed the function and added directly the line in the if block